### DISCUSSION TITLE
Check list

### DISCUSSION THREAD

**Discussion by Milo1954** [flybywiresim/aircraft#4430] [Q&A] [answered]
Is it necessary with the kilometer long check list.  i would wery much like to use the other FBW addon. But cant use these addons without thw FBW BUT can not bear to sit and wait half an hour just to start the plane up.  Isn't there an easier way Fx in NERO it only takes 2 min. to start the A320 up?

**Comment by ExampleWasTaken** [comment, marked as answer]
> 
> 
> Is it necessary with the kilometer long check list. i would wery much like to use the other FBW addon. But cant use these addons without thw FBW BUT can not bear to sit and wait half an hour just to start the plane up. Isn't there an easier way Fx in NERO it only takes 2 min. to start the A320 up?

The goal of the FBW A320neo is to create an aircraft that is as close to the real one as possible. The real one needs time to be started up so the A32NX needs it too. If don't want to spend that time, you are probably better off, using the default one.

**Reply by raknal** [reply]
i would like to locate the check list, if someone could point me to its location please

**Reply by ExampleWasTaken** [reply]
If you're talking about the checklist in the sim then you'll find the corresponding file here: `flybywire-aircraft-a320-neo\SimObjects\AirPlanes\FlyByWire_A320_NEO\Checklist\Airbus_A320neo_Checklist.xml`. If you are talking about a checklist that looks like a real world checklist (BUT IS NOT ONE), then you'll find that [here](https://cdn.discordapp.com/attachments/747571948342476850/813173610205544458/FBW_A32NX_CHECKLIST_V1.2.pdf).

**Comment by Milo1954** [comment]
thx for responding

Den ons. 14. apr. 2021 kl. 18.22 skrev ExampleWasTaken <
***@***.***>:
